scripts/tests/test_zkai_d64_commitment_consistency_method_probe.py

```
@@ -215,6 +219,39 @@ def test_validation_rejects_manifest_drift(self) -> None:
         with self.assertRaisesRegex(PROBE.CommitmentConsistencyProbeError, "proof-native parameter manifest drift"):
             PROBE.validate_probe(payload)
 
+    def test_validation_rejects_source_fixture_manifest_commitment_mismatch(self) -> None:
+        payload = PROBE.build_probe()
+        payload["proof_native_parameter_manifest"]["proof_native_parameter_commitment"] = "blake2b-256:" + "55" * 32
+        payload["proof_native_parameter_manifest_commitment"] = PROBE.blake2b_commitment(
+            payload["proof_native_parameter_manifest"],
+            "ptvm:zkai:d64:proof-native-parameter-manifest-payload:v1",
+        )
+
+        with self.assertRaisesRegex(PROBE.CommitmentConsistencyProbeError, "commitment mismatch"):
```

Ruff (0.15.12)

[warning] Use `pytest.raises` instead of unittest-style `assertRaisesRegex`

Replace `assertRaisesRegex` with `pytest.raises`

(PT027)